test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/PackageLicenseUtilitiesTests.cs

```diff
@@ -181,5 +245,43 @@ public void PackageLicenseUtility_GenerateCorrectLink()
             Assert.Equal(license, licenseText.Text);
             Assert.Equal("https://licenses.nuget.org/MIT", licenseText.Link.AbsoluteUri);
         }
+
+        private static void CreateDummyPackage(
+            string zipPath,
+            string licenseFile = "License.txt",
+            string licenseFileTargetElement = "")
+        {
+            var dir = Path.GetDirectoryName(zipPath);
+            var holdDir = "pkg";
+            var folderPath = Path.Combine(dir, holdDir);
+
+            // base dir
+            Directory.CreateDirectory(folderPath);
+
+            // create nuspec
+            var nuspec = NuspecBuilder.Create()
+                .WithFile(licenseFile, licenseFileTargetElement);
+
+            // create license file
+            var licensePath = Path.Combine(folderPath, licenseFile);
+            var licenseDir = Path.GetDirectoryName(licensePath);
+            Directory.CreateDirectory(licenseDir);
+
+            File.WriteAllText(licensePath, StreamLicenseContents);
+
+            // Create nuget package
+            using (var nuspecStream = new MemoryStream())
+            using (FileStream nupkgStream = File.Create(zipPath))
+            using (var writer = new StreamWriter(nuspecStream))
+            {
+                nuspec.Write(writer);
+                writer.Flush();
+                nuspecStream.Position = 0;
+                var pkgBuilder = new PackageBuilder(stream: nuspecStream, basePath: folderPath);
+                pkgBuilder.Save(nupkgStream);
+            }
+        }
+
+        private static string StreamLicenseContents = "I am  license";
```

Comment:
I'm not sure if this ought to be static. We have a lot tests that share recreate test class data, and singletons create potential for side effects from other tests.
Probably should also be `readonly`.
```suggestion
        private readonly string _streamLicenseContents = "I am license";
```

Reply:
probably `const`, rather than `readonly`. If the value never changes, it doesn't need to be a per-instance field.

Reply:
Const or private static readonly string would make sense for this use, I think.